bodo/tests/test_dataframe_part2.py

```diff
@@ -587,7 +587,8 @@ def test_impl(df):
     check_func(test_impl, (df,))
 
 
-@pytest.mark.df_lib
+# TODO (df_lib): properly support RangeIndex in apply
```

Comment:
Small preference for putting this TODO after the pytest.mark.df_lib on the same line so they are clearly related.